### PR TITLE
update eslintrc file format to JSON

### PR DESCRIPTION
**Pull request by davecarter**
## PR Summary

Updated `eslintrc` file format to `.json` file extension due to _deprecation_ of previous format as pointed in [eslint documentation](http://eslint.org/docs/user-guide/configuring#configuration-file-formats).

Please review @danderu @carlosvillu

### CODE REVIEW

**Comment by delacruz-dev** [comment]
Does this mean that it won't work with previous versions of eslint?

**Comment by davecarter** [comment]
This change does not break any compatibility with any previous version of `eslint`.

In addition, all components should be updated to eslint version `2.2.0` to work well with new versions of `Babel` but not further to avoid the [estraverse-fb](https://github.com/eslint/eslint/issues/5476) missing dependency issue.

An advantage to switch to `json` format is the ability to have [comments](http://eslint.org/docs/user-guide/configuring#comments-in-configuration-files) inside this file.

Hope it helps!
:smiley:

File: bin/install.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 var Validate = require('git-validate');
 
 Validate.copy('../.eslintignore', '.eslintignore', { overwrite: true });
-Validate.copy('../eslintrc', '.eslintrc', { overwrite: true });
+Validate.copy('../eslintrc', '.eslintrc.json', { overwrite: true });
```
Comment (delacruz-dev) [inline bin/install.js]: Replace with:

``` javascript
Validate.copy('../eslintrc.json', '.eslintrc.json', { overwrite: true });
```

And rename the template file in order to be consistent

**Comment by delacruz-dev** [comment]
👍

**Pull request closed without merging**